pyomo/contrib/pynumero/interfaces/ampl_nlp.py

```diff
@@ -682,11 +687,15 @@ def __init__(self, nl_file, row_filename=None, col_filename=None):
             self._name_to_con_eq_idx = {name:idx for idx,name in enumerate(self._con_eq_idx_to_name)}
             self._name_to_con_ineq_idx = {name:idx for idx,name in enumerate(self._con_ineq_idx_to_name)}
 
-            
-    def variable_names(self):
+    def primals_names(self):
         """Returns ordered list with names of primal variables"""
         return list(self._vidx_to_name)
 
+    @deprecated(msg='This method has been replaced with primals_names', version='', remove_in='6.0')
```

Comment:
Is your intent to remove this method for the next release?  If that is the case, I would recommend `version='6.0.0.dev0', remove_in='6.0'`.  Otherwise, I would either remove `remove_in`, or make it a later release (`remove_in` is the release that will no longer contain this method).

Reply:
done.